crates/protect-ffi/src/lib.rs

```diff
@@ -404,18 +386,19 @@ fn service_token_from_js_value(
     value: Option<Handle<JsValue>>,
     cx: &mut FunctionContext,
 ) -> NeonResult<Option<ServiceToken>> {
-    if let Some(service_token) = value {
-        let service_token: Handle<JsObject> = service_token.downcast_or_throw(cx)?;
+    match value {
+        Some(service_token) if is_defined(service_token, cx) => {
```

Comment:
This is the change that fixes up errors on the Rust side when the token from JS is `undefined`.